ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml

```diff
@@ -157,9 +157,9 @@
 
 - name: Set Shadow Glove Launch file inputs if real_glove = true
   set_fact:
-    launch_file_input_right: "sr_teleop_vive_polhemus.launch sim:=true side:=right vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
-    launch_file_input_left: "sr_teleop_vive_polhemus.launch sim:=true side:=left vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
-    launch_file_input_bimanual: "sr_teleop_vive_polhemus.launch sim:=true vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input_right: "sr_teleop_vive_polhemus.launch sim:=true side:=right vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```

Comment:
remove calibration_file_name var, rename calibration_file_path to boresight_calibration_file_path

Reply:
Fixed in the last commit